server/dist/wivrn.service.in

```diff
@@ -4,3 +4,17 @@ Description=OpenXR streaming server
 [Service]
 Type=simple
 ExecStart=@CMAKE_INSTALL_PREFIX@/bin/wivrn-server
+CapabilityBoundingSet=CAP_SYS_NICE
+AmbientCapabilities=CAP_SYS_NICE
+LockPersonality=true
+NoNewPrivileges=true
+PrivateTmp=true
+ProtectClock=true
+ProtectControlGroups=true
+ProtectKernelLogs=true
+ProtectKernelModules=true
+ProtectKernelTunables=true
+ProtectProc=invisible
+ProtectSystem=strict
+RemoveIPC=true
+RestrictSUIDSGID=true
```

Comment:
As we don't fork into Steam or whatever, we should be able to re-add RestrictNamespaces